Sub/Src/Mission/mission_commander.py

```diff
@@ -133,71 +143,13 @@ def _update_mission_info_callback(self, misc):
         print("[INFO]: New Mission file set as %s", self.mission_file)
         #Parse the mission file
         self.parse_mission()
-    def _command_listener(self):
-        '''
-        The thread to run update requests from the GUI to tell the mission commander
-        when it is ready to run missions and what missions to do.
 
-        Parameters:
-            N/A
-        Returns:
-            N/A
-        '''
-        while self.command_listener_thread_run:
-            try:
-                #Recieve commands from the the GUI and/or Mission Commander
-                self.mission_commander_node.spinOnce(self.movement_mode_subscriber)
```

Comment:
I think there is a merge conflict here, please try to resolve. Let me know if you need help